streampipes-client-python/streampipes/model/resource/data_lake_series.py

```diff
@@ -70,7 +59,7 @@ def from_json(cls, json_string: str) -> DataLakeSeries:
 
         Returns
         -------
-        DataLakeSeries
+        DataSeries
             Instance of `DataLakeSeries` that is created based on the given JSON string.
```

Comment:
same as above
It would be awesome if you could conduct a search for `DataLakeSeries` to catch all occurrences